go.mod

```diff
@@ -253,4 +253,4 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240604185151-ef581f913117 // indirect
 )
 
-replace github.com/moby/buildkit => github.com/okteto/buildkit v0.0.0-20241112140530-03f58aa5a54d
+replace github.com/moby/buildkit => github.com/okteto/buildkit v0.0.0-20241204113742-0ceaf11ec813
```

Comment:
Should we use the tags we create in the repo to make more obvious which version we are using? I mean, I think you can do `replace github.com/moby/buildkit => github.com/okteto/buildkit v0.17.3`

Reply:
No, because we are not creating a new tag. It´s just the branch and we need to use the commit instead to link to the branch instead of the tag

Reply:
👍  ok, but it is really hard to know which is the real version used by the CLI codebase at first glance.